### ISSUE TITLE
Replace 'Learn more' documentation links to have more descriptive names

### ISSUE THREAD

**Issue by AlekseyManetov** [camunda/camunda-modeler#5568]
### Problem you would like to solve

**Context**: During the review of [PR #1181](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/1181#issuecomment-3771743911), we identified that the current pattern of using "Learn more" name for documentation links is not optimal. While consistent with historical implementation, it lacks context and accessibility.

<img width="1520" height="366" alt="Image" src="https://github.com/user-attachments/assets/98cdd886-951a-48ac-98a0-d800867962b3" />

Problem Using "Learn more" (or "Click here") as link text is problematic for several reasons:

- **Accessibility**: Screen reader users navigating by links hear repetitive "Learn more" announcements without knowing the destination.
- **Usability**: It forces users to read the surrounding text to understand where the link leads.
- **Clarity**: It is a generic filler phrase that doesn't describe the resource.

### Proposed solution

Refactor existing documentation links across the `camunda-modeler` to use descriptive link text. The discussed recommended pattern for the names is `About [Topic]`.

**Example**

🔴 Current (Bad):

> Configure the message referenced by the task. [Learn more.](https://docs.camunda.io/docs/components/modeler/bpmn/receive-tasks/#messages)

🟢 Proposed (Good):

> Configure the message referenced by the task. [About task messages](https://docs.camunda.io/docs/components/modeler/bpmn/receive-tasks/#messages).

Note: also consider to remove dot(.) from the link name to reduce visual nose.

**Scope**: This change should be applied systematically across the `camunda-modeler` to maintain consistency.

### Alternatives considered

Leave it as is.

**Comment by barmac** [comment]
This can be implemented together with https://github.com/camunda/camunda-modeler/issues/5244 as a tooltip overhaul.
